esmvalcore/_config.py

```diff
@@ -27,6 +27,18 @@ def find_diagnostics():
 DIAGNOSTICS_PATH = find_diagnostics()
 
 
+def find_references():
+    """Try to find bibtex files in references folder."""
+    try:
+        import esmvaltool
+    except ImportError:
+        return ''
+    return os.path.join(os.path.dirname(esmvaltool.__file__), 'references')
+
+
+REFERENCES_PATH = find_references()
```

Comment:
Why not just call `find_references()` in `_task.py` instead of importing `REFERENCES_PATH`? Is this used somewhere else?

Reply:
I think it's fine to keep the configuration information together like this. At some point in the near future we will probably need to improve the way we configure esmvaltool anyway.